content/attacks/posts/2022-09-20-Wintermute.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-09-20
+custodians: Wintermute
+categories: Custodian, Trading Platform
+title: "Wintermute Incurs $160 Million Loss from Brute Force Private Key Compromise Linked to Profanity's Vulnerability"
+---
+
+## Summary
+
+On September 20, 2022, Wintermute, a London-based algorithmic market maker offering liquidity across Centralized Finance (CeFi) and Decentralized Finance (DeFi) exchanges and over-the-counter (OTC) deals, was the victim of a security breach. The exploit resulted in a loss of approximately $160 million, impacting 90 different assets including stable coins, Bitcoin, Ether, and various altcoins. The attack was executed through a brute force private key compromise. The suspected vulnerability originated from Profanity, a service Wintermute used for generating vanity addresses, despite efforts to blacklist their Profanity-associated accounts after the vulnerability became known.
+
+## Attackers
+
+The identity of the attackers remains unknown. The Ethereum address linked to the attacker and currently holding all stolen funds:
```

Comment:
add current as of date to address holding funds

Reply:
Done